### PR TITLE
spelling: corrected sentence

### PR DESCRIPTION
**Pull request by sagar-shah94**

### CODE REVIEW

**Comment by romani** [comment]
I am OK with change. @rveach your vote ?

File: src/xdocs/writingjavadocchecks.xml.vm [hunk 1]
```diff
@@ -195,7 +195,7 @@ CLASS_DEF -> CLASS_DEF [5:0]
       <p>
       As you see very small java file transforms to a huge Abstract Syntax Tree, because that is the most detailed tree including all components of the java file: classes, methods, comments, etc.
       </p>
-      <p>In most cases while developing Javadoc Check you need only parse tree of the exact Javadoc comment.
+      <p>In most cases while developing Javadoc Check you need only to parse tree of the exact Javadoc comment.
```
Comment (rnveach) [inline src/xdocs/writingjavadocchecks.xml.vm]: Please change to: `Javadoc Check, you need to only parse the tree`
comma after `Check` and `to only parse the`.

**Comment by sagar-shah94** [comment]
@rnveach @romani changes done .please approve and merge

**Comment by rnveach** [comment]
@sagar-shah94 You missed `the` after `parse`.

**Comment by sagar-shah94** [comment]
@rnveach added the after parse

**Comment by rnveach** [comment]
Fix is merged, thanks.

**Comment by sagar-shah94** [comment]
@rnveach welcome sir.

**Pull request merged**